Content.Shared/Movement/SharedMoverController.cs

```diff
@@ -152,6 +152,8 @@ protected bool UseMobMovement(PhysicsComponent body)
         {
             return body.BodyStatus == BodyStatus.OnGround &&
                    body.Owner.HasComponent<IMobStateComponent>() &&
+                   // If we're being pulled then don't mess with our velocity.
+                   (!body.Owner.TryGetComponent(out SharedPullableComponent? pullable) || !pullable.BeingPulled) &&
```

Comment:
This doesn't block mob movement breaking out of pulling; the IRelay interface is still run so that just breaks pulling and this allows us to cleanly lerp when we're being pulled.